docker/docker-compose.yml

```diff
@@ -169,17 +169,16 @@ services:
       timeout: 3s
       retries: 3
     volumes:
-      - dbdata:/var/lib/postgresql/data
+      - ${DB_DATA_VOLUME_PREVIOUS:-dbdata_pg13}:/var/lib/postgresql/data
     ports:
       - 5432
     expose:
       - 5432
 
-  # This should no longer need to be overridden for ARM
-  # processors
+  # This should no longer need to be overridden for ARM processors
   db:
```

Comment:
When I've done this sort of thing before, I've often named both the services and volumes with the postgres version to make sure they don't get mismatched. I don't think one has to do that. Just wanted to mention it.

Reply:
I think changing the service name would require updating the host in our rails config (and maybe local superset config too)

Reply:
I think with the  `container_name` set you might not need to update anything. If there are references to just the service name and that's working, that sounds fragile and I'd recommend using the container_name value instead.

Reply:
Sorry if I'm not following but I imagine folks use hostname in the config (`DATABASE_HOST=db`. I guess we could use the hostname property and rename the service

Would you propose we use something like this?
```yaml
services:
  pg16:
    container_name: hmis-warehouse-db
    image: warehousepg:16
    hostname: db
    volumes:
      - ${DB_DATA_VOLUME:-dbdata_pg16}:/var/lib/postgresql/data
    # ...
```

Or if we just use the image property, does that address the concern around mismatched containers?

Reply:
I would make sure that the hostname that people are using is unique to this app. Anybody that's using `db` or `pg16` or anything like that will have problems if they run two apps at the same time on the same docker network. My personal preference is to leave out that hostname, keep the service name pg16 and be done with it. I don't know if it matters that much. Maybe I'm the only person running more than one app at the same time.